docs/semgrep-appsec-platform/slack.md

```diff
@@ -37,23 +37,27 @@ To install the Semgrep Slack app, follow these steps:
 2. On the **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, click **Add** (or **Setup First Integration** if this is your first integration), and then select **Slack**.
 3. Click **Allow**.
 
-## Set up notifications for findings in Slack
+
+## Code and Supply Chain 
```

Comment:
I would consider reversing the order of these headers so that the hierarchy is:

- Install the Semgrep Slac App
- Set up notifications for findings in Slack
  - Code and Supply Chain
  - Secrets
- Remove notifications…
- etc.